spec/services/idv/upcase_vendor_env_vars_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe Idv::UpcaseVendorEnvVars do
+  describe '#call' do
+    before do
+      allow(Figaro.env).to receive(:proofing_vendor_profile).and_return('equifax')
+      allow(Figaro.env).to receive(:proofing_vendor_finance).and_return('aamva')
+      allow(Figaro.env).to receive(:proofing_vendor_phone).and_return('equifax')
+      stub_const 'ENV', ENV.to_h.merge(
+        'equifax_thing' => 'some value',
+        'aamva_thing' => 'other value'
+      )
+    end
+
+    it 'sets UPPER case equivalue of existing ENV var' do
```

Comment:
just a nitpick but `equal value` would've been more intuitive. I had to google what `equivalue` is

Reply:
Agree and willfix. This was copy/pasted from the spec for the `Idv::Vendor` where this code was originally.